Tasks/KubernetesManifestV0/task.json

```diff
@@ -325,16 +324,22 @@
             "label": "Docker registry service connection",
             "helpMarkDown": "Select a Docker registry service connection. Required for commands that need to authenticate with a registry.",
             "visibleRule": "action = createSecret && secretType = dockerRegistry"
+        },
+        {
+            "name": "timeout",
+            "tpye": "string",
+            "label": "Timeout for rollout status",
+            "required": false,
```

Comment:
Add default of 5m

Reply:
Default value could be set to 0 where you would take the kubectl's default value

Reply:
Keep default as whatever value we are using for the stability checks in the task today. Setting any other value will be an unexpected change for whoever is already using the task

Reply:
By default, if the deployment is not successful even after 10 min, we get an error stating that the progress deadline exceeded. Setting the default value as 0 means that the watch on rollout status would be indefinitely true, and thus after 10 min it would show the same error.